packages/client/modules/meeting/components/AddNewPokerTemplate.tsx

```diff
@@ -28,13 +30,43 @@ const AddPokerTemplateLink = styled(LinkButton)({
 
 interface Props {
   gotoTeamTemplates: () => void
-  pokerTemplates: AddNewPokerTemplate_pokerTemplates
-  teamId: string
+  pokerTemplatesRef: AddNewPokerTemplate_pokerTemplates$key
+  viewerRef: AddNewPokerTemplate_viewer$key
+  teamRef: AddNewPokerTemplate_team$key
+  setShowUpgradeDetails: (showUpgradeDetails: boolean) => void
```

Comment:
-1 it's only ever called with `true` so it should be `showUpgradeDetails: () => void` instead

Reply:
It's called with `false` in the modal: https://github.com/ParabolInc/parabol/blob/2eaee63fe719d48b781de220e40cc88ba395c8a4/packages/client/modules/meeting/components/PokerTemplateModal.tsx#L85. In this file, it's only ever called with `true`, so I could change it to `showUpgradeDetails: (showUpgradeDetails: true) => void`?

Reply:
It should be changed to `showUpgradeDetails: () => void` in all the components who receive the callback. The fact it is a true/false state in PokerTemplateModal is not relevant for the other components.

Reply:
`showUpgradeDetails` is the boolean value, so do you mean it should be changed to `setShowUpgradeDetails: () => void`? That wouldn't work in this file ,`AddNewPokerTemplate`, as we're passing true into `setShowUpgradeDetails` on line 80.

Or are you thinking that in `PokerTemplateModal` I do the following:

```

  const displayUpgradeDetails = () => {
    setShowUpgradeDetails(true)
  }

  const hideUpgradeDetails = () => {
    setShowUpgradeDetails(false)
  }

  useEffect(() => {
    if (showUpgradeDetails) hideUpgradeDetails()
  }, [activeTemplate])
```

and pass `displayUpgradeDetails` down to the child components? Then the prop type would be `displayUpgradeDetails: () => void`

Reply:
Yes, the second part. Sorry if that wasn't clear. Create a function you pass down to the other dependencies which is `() => void`.

Reply:
Makes sense! I've pushed an update with that change 👍